projects/configs/tag-operations/base.yaml

```diff
@@ -0,0 +1,660 @@
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/x-ray-grid/00181166
+#https://dicom.innolitics.com/ciods/x-ray-3d-craniofacial-image/x-ray-3d-craniofacial-acquisition/00189507/00181166
+#https://dicom.innolitics.com/ciods/x-ray-3d-angiographic-image/x-ray-3d-angiographic-acquisition/00189507/00181166
+#https://dicom.innolitics.com/ciods/breast-projection-x-ray-image/breast-projection-x-ray-image-multi-frame-functional-groups/52009229/00189555/00181166
+- name: "Focal Spot"
+  group: 0x0018
+  element: 0x1190
+  op: "keep"
+#https://dicom.innolitics.com/ciods/ct-image/ct-image/00181190
+#https://dicom.innolitics.com/ciods/ct-image/ct-image/00189360/00181190
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/x-ray-acquisition/00181190
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/x-ray-acquisition/00181190
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/x-ray-generation/00181190
+#https://dicom.innolitics.com/ciods/digital-mammography-x-ray-image/x-ray-generation/00181190
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/x-ray-generation/00181190
+#https://dicom.innolitics.com/ciods/breast-projection-x-ray-image/enhanced-mammography-image/00181190
+#https://dicom.innolitics.com/ciods/x-ray-3d-angiographic-image/x-ray-3d-angiographic-acquisition/00189507/00181190
+- name: "Receive Coil Name"
+  group: 0x0018
+  element: 0x1250
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00181250
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189042/00181250
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189042/00181250
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189042/00181250
+- name: "Transmit Coil Name"
+  group: 0x0018
+  element: 0x1251
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00181251
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189049/00181251
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189049/00181251
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189049/00181251
+- name: "Flip Angle"
+  group: 0x0018
+  element: 0x1314
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00181314
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189112/00181314
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189112/00181314
+- name: "Acquisition Device Processing Description"
+  group: 0x0018
+  element: 0x1400
+  op: "keep"
```

Comment:
I think should be in MRI.yaml

Reply:
[acc3b3c](https://github.com/SAFEHR-data/PIXL/pull/498/commits/acc3b3cebbbbb8987bc01783ccbc89fee30d51bf)